subsys/bluetooth/audio/Kconfig.pbp

```diff
@@ -0,0 +1,16 @@
+# Bluetooth Audio - Public Broadcast Profile (PBP) options
+#
+# Copyright 2023 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config BT_PBP
+	bool "Public Broadcast Profile [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  Enabling this will enable PBP.
+
+module = BT_PBP
+module-str = "Public Broadcast Profile"
+source "subsys/bluetooth/common/Kconfig.template.log_config_bt"
```

Comment:
Not sure exactly why, but all other profiles etc. logging are defined in `subsys/bluetooth/Kconfig.logging` - Makes sense to be consistent IMO